dts/bindings/sensor/maxim,ds18b20.yaml

```diff
@@ -14,3 +14,8 @@ properties:
     description: |
         Resolution of the temperature reading in bit.
         Valid configurations are from 9 to 12 bits.
+  ds18s20:
```

Comment:
instead of a boolean property, I think it would be more accurate to add another compatible